MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -1392,12 +1360,12 @@ private bool IsAudioEligibleForDirectPlay(MediaSourceInfo item, long maxBitrate,
             var requestedMaxBitrate = maxBitrate > 0 ? maxBitrate : 1000000;
 
             // If we don't know the bitrate, then force a transcode if requested max bitrate is under 40 mbps
-            var itemBitrate = item.Bitrate ??
-                40000000;
+            var itemBitrate = item.Bitrate ?? 40000000;
```

Comment:
what the heck about 40mbps... doesn't make any sense IMHO

Reply:
:man_shrugging: Not mine ;)